README.md

```diff
@@ -1,2 +1,100 @@
 # Rest7Contract
-Repository to discuss the new REST API contract for DSpace 7 https://wiki.duraspace.org/display/DSPACE/DSpace+7+UI+Working+Group
+Repository to discuss the new REST API contract for DSpace 7 <https://wiki.duraspace.org/display/DSPACE/DSpace+7+UI+Working+Group>
+
+## Use of the HTTP Verbs and HTTP Response CODE
+
+### On collection of resources endpoints
+- POST
+Adds a new element to the collection.
+
+- GET
+Returns the first page of the resources in the collection
+
+### On single resource endpoints
+- GET
+Returns a single entity.
+
+- HEAD
+Returns whether the item resource is available.
+
+- PUT
+Replaces the state of the target resource with the supplied request body.
+
+- PATCH
+Similar to PUT but partially updating the resources state.
+
+- DELETE
+Deletes the resource exposed.
+
+### On sub-path of a single resource endpoint)
+- GET
+Returns the state of the association resource
+
+- PUT
+Binds the resource pointed to by the given URI(s) to the resource. Return 400 Bad Request if multiple URIs were given for a to-one-association
+
+- POST
+Only supported for collection associations. Adds a new element to the collection.
+
+- DELETE
+Unbinds the association. Return 405 Method Not Allowed if the association is non-optional
+
+### Error codes
```

Comment:
you are right, these are enough general situation to be noted in this introduction instead to be hidden in the single endpoint documentation